helm/deletion-blocker-operator/values.schema.json

```diff
@@ -156,7 +156,15 @@
                     "type": "boolean"
                 },
                 "capabilities": {
-                    "type": "object"
+                    "type": "object",
+                    "properties": {
+                        "drop": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            }
+                        }
+                    }
                 },
```

Comment:
This change is not related to the purpose of the PR. I ran `helm-schema-gen` to find anything I could update about the schema, to please our stupid schema check, and found this missing chunk.